src/dc_imex.rs

```diff
@@ -1055,47 +1056,32 @@ unsafe fn export_self_keys(context: &Context, dir: *const libc::c_char) -> bool
 unsafe fn export_key_to_asc_file(
```

Comment:
Because you're cleaning up this function I'd really like new unittests covering it.  I think it's good to increase test coverage as we cleanup functions, I try to do this as well.  There are already tests with keys so most groundwork for testing infrastructure probably exists already.

Reply:
I'll take a look. This function does IO, so preparations may be long.

Reply:
Added test.